metricbeat/module/mongodb/dbstats/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"mysql",
+        "name":"status",
+        "rtt":44269
+    },
+    "mongodb":{
+        "dbstats":{
+            "example": "dbstats"
```

Comment:
An example doc should be generated here with `make generate-json`. But that can still be done after merging the PR.